lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -144,6 +144,17 @@ describe('CategoryRenderer', () => {
 
     const audits = categoryDOM.querySelectorAll('.lh-audit');
     assert.equal(audits.length, category.auditRefs.length, 'renders correct number of audits');
+
+    assert.equal(categoryDOM.querySelector('.lh-gauge__wrapper--plugin'), null);
+  });
+
+  it('plugin category has plugin badge', () => {
+    const category = JSON.parse(
+      JSON.stringify(sampleResults.reportCategories.find(c => c.id === 'seo')));
+    // Any category not of the core
```

Comment:
not sure what this means :)

```
Be bloody, bold, and resolute; laugh to scorn
The power of core, for none of plugin born
Shall harm the test.
```

Reply:
i must have left for lunch mid-comment :)

just gonna delete this.